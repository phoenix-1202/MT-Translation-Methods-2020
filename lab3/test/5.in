funWithManyOperationsAndBrackets :: Int -> Int -> Int
funWithManyOperationsAndBrackets a b = (((* (min ((a)) b) + (max a b) 5) / (((12)))))