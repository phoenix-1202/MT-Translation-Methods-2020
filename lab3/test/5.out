#include <bits/stdc++.h>;
using namespace std;

int funWithManyOperationsAndBrackets(int arg1, int arg2) {
return (((min(arg1, arg2) + max(arg1, arg2)) * 5) / 12);
}

//EXIT